Dockerfile_windows

```diff
@@ -0,0 +1,7 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
```

Comment:
Do we have to rely on this?

Reply:
So, this is complicated. We cannot use `nanoserver` as it doesn't provide `Netapi32.dll` [link](https://github.com/docker-library/golang/issues/348)

Also it means for windows what are the base image and what system we try to run it on. Manifest should have addition `os.version` field, and ideally we should provide multiple windows images. Not sure if newer windows can run older images, as github actions had issues with running image based on older tag.

Due to above, I'm not sure if this should be run for release branches, as those are still experimental images

Reply:
```
➜  ~ docker manifest inspect public.ecr.aws/sumologic/sumologic-otel-collector-dev:0fa281c6c04691e937f403218b09de6e91f04d29
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1093,
         "digest": "sha256:719c9ffed95ff6eda12c6564e70c1c025352dacb376f5dfc01a37cdc87f6953d",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1093,
         "digest": "sha256:8a04067bc357fee33f1e11b468e408ed26950f0ea7382d47ccc6b3614f2150d9",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1788,
         "digest": "sha256:914c7d0ee0927e2d065f6b74f03ab036e9758afa76cb09d54cf99340dbb9609b",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.20348.2322"
         }
      }
   ]
}
```

Working on building multitple images, as this one is not working on my eks windows cluster :sweat_smile: